.eslintrc.cjs

```diff
@@ -0,0 +1,43 @@
+module.exports = {
+  env: {
+    node: true,
+  },
+  root: true,
+  parser: "@typescript-eslint/parser",
+  plugins: ["@typescript-eslint", "jest", "jsdoc"],
+  extends: [
+    "eslint:recommended",
+    "prettier",
+    "plugin:prettier/recommended",
+    "plugin:@typescript-eslint/recommended",
+  ],
+  ignorePatterns: ["build/*"],
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: "module",
+  },
+  rules: {
```

Comment:
Can we add rules to omit formatting lint rules from prettier or at least not make them lint errors?

per our discussion from a few months ago https://mongodb.slack.com/archives/C05CE6UBGG5/p1696254787545609